SSO-Auth/Config/configPage.html

```diff
@@ -1,15 +1,272 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
+  <head>
     <title>SSO</title>
-</head>
-<body>
-    <div data-role="page" class="page type-interior pluginConfigurationPage esqConfigurationPage">
-        <div data-role="content">
-            <div class="content-primary">
-                <a href="https://github.com/9p4/jellyfin-plugin-sso/blob/main/README.md">Review the documentation. This plugin is configured via the API.</a>
+  </head>
+  <body>
+    <div
+      id="sso-config-page"
+      data-role="page"
+      class="page type-interior pluginConfigurationPage esqConfigurationPage"
+      data-controller="__plugin/SSO-Auth.js"
+    >
+      <div data-role="content">
+        <div class="content-primary">
+          <a
+            href="https://github.com/9p4/jellyfin-plugin-sso/blob/main/README.md"
+            >Review the documentation. This plugin is configured via the API.</a
+          >
+
+          <form id="sso-load-config" class="esqConfigurationForm">
+            <div class="selectContainer">
+              <label class="selectLabel" for="selectProvider"
+                >Choose OIDC Provider to load configuration for:</label
+              >
+              <select
+                is="emby-select"
+                id="selectProvider"
+                name="selectProvider"
+                class="emby-select-withcolor emby-select"
+              ></select>
+              <div class="selectArrowContainer">
+                <div style="visibility: hidden; display: none">0</div>
+                <span
+                  class="selectArrow material-icons keyboard_arrow_down"
+                  aria-hidden="true"
+                ></span>
+              </div>
             </div>
+
+            <button
+              id="LoadProvider"
+              is="emby-button"
+              type="button"
+              class="raised button-submit block emby-button"
+            >
+              <span>Load Provider</span>
+            </button>
+
+            <button
+              id="DeleteProvider"
+              is="emby-button"
+              type="button"
+              class="raised button-submit block emby-button"
+            >
+              <span>Delete Provider</span>
+            </button>
+          </form>
+
+          <form id="sso-new-oidc-provider" class="esqConfigurationForm">
+            <div class="verticalSection verticalSection-extrabottompadding">
+              <div class="inputContainer">
+                <label
+                  class="inputLabel inputLabelUnfocused"
+                  for="OidProviderName"
+                  >Name/ID of OID Provider:</label
```

Comment:
Should be "Name of OpenID Provider:"

Reply:
https://github.com/9p4/jellyfin-plugin-sso/pull/18/commits/3bc8cc9a85dd51bd1f18dfb597b16a99d6197a2f#diff-d1cae9c0a00116bf22523395cbfe6b7f5d9603ef680a7202c05e4a53bc3b05cfL59-R61 resolved